release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -402,6 +395,17 @@ affinity scheduler must be manually configured.
 See xref:../nodes/scheduling/nodes-scheduler-node-affinity.adoc#nodes-scheduler-node-affinity[Controlling Pod placement on nodes using node affinity rules]
 for more information.
 
+[id="ocp-4-5-virtual-hosted-buckets-s3"]
+==== Virtual hosted buckets for custom S3 endpoints
+
+Virtual hosted buckets are now supported to deploy clusters in new or hidden AWS regions.
+
+[id="ocp-4-5-node-pull-secrets"]
+==== Node pull credentials during build and imagestream import
+
+Builds and Imagestream imports will automatically use the pull secret used to install the cluster if a pull secret is not explicitly set. Developers will not need to copy this pull secret into their namespace.
```

Comment:
Do you want to match capitalization for "build" and "imagestream" between the h4 and first sentence?

Reply:
Instead of "will not" in the second sentence, how about "do not"? Just a preference.